src/Error/Debug/dumpHeader.html

```diff
@@ -211,6 +225,27 @@
   }
 }
 
+function attachCopyButton(nodes) {
+  nodes.forEach(function (container) {
+    var copy = doc.createElement('a');
+    copy.classList.add('cake-dbg-copy');
+    copy.setAttribute('href', '#')
+    copy.setAttribute('title', 'Copy contents of debug output');
+    copy.appendChild(doc.createTextNode('Copy'));
+
+    // Add copy behavior
+    copy.addEventListener('click', function (event) {
+      event.preventDefault();
+      event.stopPropagation();
+
+      // Chop off last 4 to exclude copy button text.
+      navigator.clipboard.writeText(container.innerText.substring(0, container.innerText.length - 4));
```

Comment:
```suggestion
      navigator.clipboard.writeText(container.textContent.substring(0, container.textContent.length - 4));
```

`innerText` won't include whitespaces and visually hidden elements.